core/src/main/java/org/apache/carbondata/core/metadata/converter/ThriftWrapperSchemaConverterImpl.java

```diff
@@ -110,6 +108,7 @@
         return org.apache.carbondata.format.Encoding.BIT_PACKED;
       case DIRECT_DICTIONARY:
         return org.apache.carbondata.format.Encoding.DIRECT_DICTIONARY;
+      case DICTIONARY:
       default:
         return org.apache.carbondata.format.Encoding.DICTIONARY;
```

Comment:
Actually I think here we should check it more strictly. If we do not recognize it, we should throw exception.